SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -181,40 +181,40 @@ BerserkAI = 1
 
 # Combat textbox remains open after right-clicking on an opponent (lets you see some spells duration etc.)
 # Изменить поведение информационного окна при клике правой кнопкой мыши на вражеском существе (окно остается открытым)
 # Die Kampf-Textbox bleibt geöffnet nach Rechtsklick auf einen Gegner (Es erlaubt dem Spieler die Dauer einiger Zaubersprüchen zu sehen usw.)
-# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)  
+# Το πλαίσιο κειμένου στη μάχη παραμένει ανοικτό μετά το δεξί κλικ σε αντίπαλο (σας επιτρέπει να δείτε τη διάρκεια που έχουν ορισμένα ξόρκια κλπ.)
 # W trakcie walki okna informacyjne po kliknięciu prawym klawiszem myszki są dostępne (nie znikają) także dla jednostek wroga (umożliwią np. sprawdzenie długości działania niektórych zaklęć).
-CombatRMBTextbox = 1
+CombatRMBTextbox = 0
 
 # Shows alpha-numeric combat coordinates when you mouse over various hexes during combat
 # При наведении и перемещении мыши отображать координаты гексов поля боя
 # Es zeigt alphanumerische Kampf-Koordinaten, wann man die Maus über verschiedene Hexadezimalschreibweisen während der Kampf bewegt
 # Εμφανίζει αλφαριθμητικές συντεταγμένες μάχης όταν μετακινείτε τον κέρσορα πάνω από διάφορα τετραγωνάκια κατά τη διάρκεια της μάχης
 # Pokazuje w formie alfanumerycznej koordynaty walki gdy podczas potyczki przesuwasz wskaźnik myszki nad heksami.
-CombatCoordinates = 1
+CombatCoordinates = 0
 
 # Show creature's hitpoints when you mouse over them in combat
 # При наведении мыши на существо отображать очки его здоровья в активном логе битвы
 # Es zeigt die Lebenspunkte einer Kreatur, wann man die Maus über sie während der Kampf bewegt
 # Εμφανίζει τους πόντους ζωής ενός πλάσματος όταν περνάτε τον κέρσορα από πάνω του στη διάρκεια της μάχης
 # Pokazuje pozostałe punkty życia stworzenia, gdy podczas walki najedziesz na niego kursorem.
-ShowHitPoints = 1
+ShowHitPoints = 0
 
 # Show creature move order on mouse hover during combat
 # NOTE: still experimental and not completely accurate
 # При наведении мыши на существо показывать порядковый номер в списке приоритетов очередности хода на поле боя
-# ПРИМЕЧАНИЕ: опция экспериментальная и не откалиброванная (возможны неточности) 
+# ПРИМЕЧАНИЕ: опция экспериментальная и не откалиброванная (возможны неточности)
 # Es zeigt die Bewegungsreihenfolge der Kreatur, wann man die Maus über sie während der Kampf hält
 # ANMELDUNG: Es ist noch vorläufig und nicht absolut genau
 # Εμφανίζει τη σειρά κίνησης του πλάσματος όταν περνάτε το ποντίκι από πάνω του στη διάρκεια της μάχης
-# ΣΗΜΕΙΩΣΗ: Είναι ακόμα σε πειραματικό στάδιο και δεν είναι ακόμα απολύτως ακριβές 
+# ΣΗΜΕΙΩΣΗ: Είναι ακόμα σε πειραματικό στάδιο και δεν είναι ακόμα απολύτως ακριβές
 # Podczas walki tuż przy kursorze pokazuje kolejność ruchu stworzenia.
 # UWAGA: do wciąż eksperymentalna opcja i nie do końca działająca
-ShowMoveSequence = 1
+ShowMoveSequence = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.